### PR TITLE
[FIX] general 

### PR DESCRIPTION
**Pull request by oliveralka**
I will add some more in the following days.

### CODE REVIEW

File: Handout/gettingstarted.tex [hunk 1]
```diff
@@ -549,7 +553,7 @@ \subsubsection{A minimal workflow}
 \subsubsection{Digression: Working with chemical structures}
 Metabolomics analyses often involve working with chemical structures. Popular cheminformatic toolkits such as RDKit~\cite{rdkit} or CDK~\cite{cdk} are available as KNIME plugins and allow us to work with chemical structures directly from within KNIME. In particular, we will use KNIME and RDKit to visualize a list of compounds and filter them by predefined substructures. Chemical structures are often represented as SMILES (\textbf{S}implified \textbf{m}olecular \textbf{i}nput \textbf{l}ine \textbf{e}ntry \textbf{s}pecification), a simple and compact way to describe complex chemical structures as text. For example, the chemical structure of L-alanine can be written as the SMILES string C[C@H](N)C(O)=O. As we will discuss later, all OpenMS tools that perform metabolite identification will report SMILES as part of their result, which can then be further processed and visualized using RDKit and KNIME.
 
-\begin{figure}
+\begin{figure}s
```
Comment (jpfeuffer) [inline Handout/gettingstarted.tex]: does that work?

File: Handout/gettingstarted.tex [hunk 2]
```diff
@@ -577,26 +581,28 @@ \subsubsection{Digression: Working with chemical structures}
 \end{itemize}
 
 
-\subsubsection{Advanced topic: Meta nodes}
+\subsubsection{Advanced topic: Metanodes}
```
Comment (jpfeuffer) [inline Handout/gettingstarted.tex]: Unfortunately, I think they are called Components now..

Comment (jpfeuffer) [reply Handout/gettingstarted.tex]: Well the wrapped Metanodes at least. We need to double-check

File: Handout/labelfree.tex [hunk 3]
```diff
@@ -50,23 +52,23 @@ \subsection{Peptide Identification}
 The fasta file we used before already contains such a decoy database.
 For OpenMS to know which OMSSA PSM came from which part of the file (i.e. target versus decoy), we have to index the results.
 To this end, extend the workflow with a \KNIMENODE{PeptideIndexer} node \menu{Community Nodes > OpenMS > ID Processing}.
-This node needs the idXML as input as well as the database file.
+%Ref to Fig
+This node needs the idXML as input as well as the database file (see Fig fig:id_fdr).
```
Comment (jpfeuffer) [inline Handout/labelfree.tex]: This seems to be a TODO

File: Handout/gettingstarted.tex (hunk 2 above)
Comment (oliveralka) [reply Handout/gettingstarted.tex]: I think it is still a Metanode... seems there are Metanodes and Components now.

File: Handout/labelfree.tex (hunk 3 above)
Comment (oliveralka) [reply Handout/labelfree.tex]: I think this is shown in the figure...

Comment (jpfeuffer) [reply Handout/labelfree.tex]: But the figure is not referenced correctly, right? I think there is a \ref missing

File: Handout/gettingstarted.tex (hunk 2 above)
Comment (jpfeuffer) [reply Handout/gettingstarted.tex]: Yes, the wrapped ones (now components) are the ones that have an own config dialog and views. They also have a separate variable environment and can be shared and reused on the Workflow Hub.

The normal ones are just bundling nodes such that the workflow looks cleaner.

**Comment by oliveralka** [comment]
@jpfeuffer - should be resolved now - please let me know if anything else is missing.

**Pull request merged**